source/frontend/src/utils/test-utils.tsx

```diff
@@ -22,23 +22,25 @@ export const mockKeycloak = (
     roles?: string[];
     organizations?: number[];
     authenticated?: boolean;
+    userInfo?: any;
   } = {},
 ) => {
-  const { claims, roles, organizations, authenticated = true } = props;
+  const { claims, roles, organizations, authenticated = true, userInfo } = props;
   // mock what would be returned by keycloak userinfo endpoint
-  const userInfo = {
+  const defaultUserInfo = {
     organizations: organizations ?? [1],
     client_roles: [...(claims ?? []), ...(roles ?? [])] ?? [],
     email: 'test@test.com',
     name: 'Chester Tester',
+    idir_user_guid: '7DB280070D474EF0BB46C365A4B95A73',
```

Comment:
The secret scanning action is not happy with this string - please change it to something else; e.g. "foobar"